bond_app/routes.py

```diff
@@ -228,7 +228,9 @@ def clear_expired_datastore_entries():
 def get_status():
     sam_base_url = config.get('sam', 'BASE_URL')
 
-    providers = {section_name: FenceApi(config.get(section_name, 'FENCE_BASE_URL'))
+    providers = {section_name: FenceApi(config.get(section_name, 'FENCE_BASE_URL'),
+                                        config.get(section_name, 'IGNORE_CERTIFICATES')
+                                        if config.has_option(section_name, 'IGNORE_CERTIFICATES') else False)
```

Comment:
qq, does config.has_option retrieve the value if its present, or only check for the existence of the option?
e.g. if we have IGNORE_CERTIFICATES specified to False, will has_option still return true?

Reply:
It only checks for the existence of the option. So if IGNORE_CERTIFICATES is False, `config.has_option` will return True, and we'll pass along the specified config value

Reply:
Maybe not a big deal, but I think that's pretty confusing. I'd expect setting IGNORE_CERTIFICATES to False to do that. I have a mild preference for fixing this up if we're going to keep this code after the fence fixes their certs.